packages/react-layouts/src/Tabs/Tab/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Quid, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @flow
+import * as React from 'react';
+import css from '@emotion/css/macro';
+import styled from '@emotion/styled/macro';
+import { textStyles, withFallback as wf, sizes } from '@quid/theme';
+
+export type Props = {
+  children: React.Node,
+  id: string,
+  isActive: boolean,
+  onClick: (SyntheticEvent<Element>) => void,
```

Comment:
I think you may want to use HTMLElement rather than Element. The latter also includes `documentElement` and other unwanted stuff